helm/values.yaml

```diff
@@ -14,11 +14,26 @@ storage:
     repository: ghcr.io/rancher-sandbox/sbombastic/storage
     tag: v0.1.0-alpha1
     pullPolicy: IfNotPresent
-  replicas: 3
+  replicas: 1
```

Comment:
Please, file an issue about whether we should keep using `Deployment` to run the storage component, as opposed to `StatefulSet`. I left a comment about that above.